### ISSUE TITLE
Ruby 1.8 not available

### ISSUE THREAD

**Issue by StephanMeijer** [asdf-vm/asdf-ruby#65]
```bash
➜  folder git:(development) ✗ asdf list-all ruby | grep 1.8
jruby-9.1.8.0
1.8.5-p115
1.8.6-p114
1.8.6-p383
1.8.6-p388
1.8.6-p398
1.8.6-p399
1.8.6-p420
1.8.7-p174
1.8.7-p248
1.8.7-p249
1.8.7-p299
1.8.7-p302
1.8.7-p330
1.8.7-p334
1.8.7-p352
1.8.7-p357
1.8.7-p358
1.8.7-p370
1.8.7-p371
1.8.7-p374
2.1.8
➜  folder git:(development) ✗ asdf install ruby 1.8.7-p374
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   168  100   168    0     0    238      0 --:--:-- --:--:-- --:--:--   238

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

**Comment by domcorvasce** [comment]
_asdf-ruby_ download the various Ruby versions using https://github.com/rbenv/ruby-build.
Currently, the oldest Ruby version available is **1.8.5-p52**  but you can access other versions from the official repository: http://cache.ruby-lang.org/pub/ruby/1.8.
You can try to submit a PR to **ruby-build** repository adding the version you are interested or open an issue so someone else can do it.

**Comment by Stratus3D** [comment]
Yes, this is something ruby-build maintainers must address. This isn't something we can address in asdf-ruby.
